cppQML/.vscode/tasks.json

```diff
@@ -120,9 +210,10 @@
             ],
             "dependsOrder": "sequence",
             "dependsOn": [
-                "update-qt-ini-arm64",
                 "kill-redirect-qml-debugger",
                 "deploy-torizon-arm64",
+                "update-qt-ini-arm64",
```

Comment:
Will a new Qt Creator instance be opened every time this task is executed?

Reply:
@andreriesco Yes, this is the best and more quick way to do it, also the most safe because does not cancel any work unsaved from the user without their consent.

Reply:
Actually I think we can include the `update-qt-ini-arm64` on the `dependsOn` of `open-in-qt-creator-debug-arm64`, and remove both of this from this `dependsOn`.
In that way we will not be closing the Qt Creator from the user and it will not be necessary to open many instances of Qt Creator. The user experience will be -> open Qt Creator for the desired arch -> run this task -> make modifications on the project in this Qt Creator -> run this task again -> make modifications on the project in this Qt Creator -> run this task again -> ...

Reply:
Just like how it is today, but opening Qt Creator with the right settings for the desired arch

Reply:
hmm, ok make sense, let me change it